core/scripts/chaincli/command/keeper/launch.go

```diff
@@ -47,7 +47,7 @@ var launchAndTestCmd = &cobra.Command{
 
 func init() {
 	launchAndTestCmd.Flags().BoolP("withdraw", "w", true, "Specify if funds should be withdrawn and upkeeps should be canceled")
-	launchAndTestCmd.Flags().BoolP("bootstrap", "b", true, "Specify if launching bootstrap node is required. Default listen ports(5688, 8000) are used, if you need to use custom ports, please use bootstrap command")
+	launchAndTestCmd.Flags().BoolP("bootstrap", "b", false, "Specify if launching bootstrap node is required. Default listen ports(5688, 8000) are used, if you need to use custom ports, please use bootstrap command")
```

Comment:
Any specific reason of changing this to false? I thought having one command which covers both bootstrap and launchAndTest is smoother for people. 

May worth adding a note to the wiki if we update this behavior: https://github.com/smartcontractkit/chainlink/tree/develop/core/scripts/chaincli Thanks.

Reply:
I think this change is consistent with the README

Reply:
I see, you are right. In the README, we have bootstrap as an independent step, and people are still following the steps in that order.